.github/workflows/function-composition-bench.yml

```diff
@@ -184,6 +186,105 @@ jobs:
     - name: Delete
       run: KUBECONFIG=/etc/kubernetes/admin.conf ./function-images/tests/chained-function-eventing/manifests/delete.sh
 
+    - name: Cleaning
+      if: ${{ always() }}
+      run: ./scripts/github_runner/clean_cri_runner.sh stock-only
+
+  bench-video-compose:
+    name: Video Processing (docker-compose)
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup grpcurl
+      run: |
+        wget -qO- https://github.com/fullstorydev/grpcurl/releases/download/v1.8.1/grpcurl_1.8.1_linux_x86_64.tar.gz | sudo tar -C /usr/bin/ -xz grpcurl
+
+    - name: start multi-container program with docker-compose
+      run: |
+        cd ./function-images/tests/video-bench/
+        docker-compose up &> log_file &
```

Comment:
I copied this from elsewhere, we had this discussion already a while ago: docker-compose cannot be blocking otherwise the test never finishes. For this reason we pipe to a log file and make it run in the background.